### PR TITLE
Add simple policy gradient comparison

### PR DESCRIPTION
**Pull request by corentinlger**
- Added a file to train a simple policy gradient algorithm (update the policy network every n_steps and reset the environment)
- Added the hydra config files in a separate folder
- Modified the plotting function to accept different experiments
- Added the results with 32 random seeds (and the same network parameters than for DDQN)

The code will surely need some refactoring (it doesn't follow the same style kaqn.py and the plot.py could be improved. The results look quite promising compared to DDQN.

Results with KAN and MLP networks for 32 seeds (no regularization used for KAN):

Simple_PG_results

Results with KAN and MLP networks for 32 seeds (with regularization for KAN), it doesn't seem to change the results a lot with the chosen hyperparameters:

Simple_PG_results_w_reg

I am not sure to understand why MLP works best here compared to other experiment (the training code should have only changed for KAN). 

Also the default ```lamb``` value was set to 0.0 in the DDQN training file, does this mean you didn't use the regularization in the loss during your experiments @riiswa  ?

### CODE REVIEW

**Comment by riiswa** [comment]
oh yes you're right, and finally I think is not necessary for our purpose, because here we only need accurary and not learning interpretable policy

**Comment by corentinlger** [comment]
Indeed ! Do you want me to do some changes on the current PR ? 

I added reward to go in the algorithm (and didn't use reg, like in DDQN exexperiment) and now the learning curves look like that:  

Simple_PG_results
